modules/gapi/include/opencv2/gapi/gcomputation.hpp

```diff
@@ -259,6 +259,9 @@ class GAPI_EXPORTS_W GComputation
      */
     void apply(GRunArgs &&ins, GRunArgsP &&outs, GCompileArgs &&args = {});       // Arg-to-arg overload
 
+    /// @private -- Exclude this function from OpenCV documentation
+    GAPI_WRAP GRunArgs apply(GRunArgs &&ins, GCompileArgs &&args = {});
```

Comment:
Why not make it shadow?

Reply:
hmmm, by moving it to `shadow_gapi` class with the same name ?

Reply:
Why class? Just function `apply`. Comment says `Exclude this function from OpenCV documentation` - does is mean that it's not used in C++ but defined only for Python?

Reply:
It is method

Reply:
`apply` implementation requires internal headers such as `gcomputation_priv.hpp`, so it can't be moved.
I hope it can be entirely implemented using `PythonC` and put in `pyopencv_gapi.hpp` in next PR's